### PR TITLE
npmpublish.yml

### PR DESCRIPTION
**Pull request by ericramos1980**
name: Node.js Package

on:
  release:
    types: [created]

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v1
        with:
          node-version: 12
      - run: npm ci
      - run: npm test

  publish-npm:
    needs: build
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v1
        with:
          node-version: 12
          registry-url: https://registry.npmjs.org/
      - run: npm ci
      - run: npm publish
        env:
          NODE_AUTH_TOKEN: ${{8e8d00fe-5b2e-4c0f-8d0f-fc5a8c15e40c}}

  publish-gpr:
    needs: build
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v1
        with:
          node-version: 12
          registry-url: https://npm.pkg.github.com/
          scope: '@ericramos1980
      - run: npm ci
      - run: npm publish
        env:
          NODE_AUTH_TOKEN: ${{ 1e4d0db522cd0685f244c977f102905ce0c871a9}}

# What / Why

> n/a

## References

* n/a

### CODE REVIEW

File: .github/workflows/npmpublish.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - run: npm ci
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{8e8d00fe-5b2e-4c0f-8d0f-fc5a8c15e40c}}
```
Comment (ljharb) [inline .github/workflows/npmpublish.yml:30]: this token is now compromised; you may want to rotate it.

File: .github/workflows/npmpublish.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://npm.pkg.github.com/
+          scope: '@ericramos1980
+      - run: npm ci
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{ 1e4d0db522cd0685f244c977f102905ce0c871a9}}
```
Comment (ljharb) [inline .github/workflows/npmpublish.yml:45]: this one too

File: .github/workflows/npmpublish.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://npm.pkg.github.com/
+          scope: '@ericramos1980
+      - run: npm ci
```
Comment (ljharb) [inline .github/workflows/npmpublish.yml:42]: you have an extra apostrophe here.

**Review by ljharb — COMMENTED** [review]
are you trying to get the npm cli published under your personal scope?

**Comment by darcyclarke** [comment]
I believe this PR was mistakenly open, unless there's something I'm missing here. To @ljharb's points, it looks like you're going to have to rotate those NPM auth tokens now that you've shared them here.

I'll be closing this PR now.

**Comment by armano2** [comment]
@darcyclarke it seems that this is a bot or his account got hacked, he opened alot of almost same PR across a lot of repositories, that includes git, npm, codecov, nginx, electron list go on

in some of them he is also is trying to change/replace licenses with his name or create some weird files

eg.
https://github.com/nginx/nginx/pull/49/files
https://github.com/npm/cli/pull/785
https://github.com/git/git/pull/704
https://github.com/gitcoinco/web/pull/5911
https://github.com/indutny/bn.js/pull/240
https://github.com/ethjs/ethjs/pull/35
https://github.com/kubernetes/website/issues/16811

this list goes on

**Pull request closed without merging**